api/routers/products.py

```
@@ -0,0 +1,97 @@
+    excel_file = io.BytesIO()
+    wb.save(excel_file)
+    excel_file.seek(0)
+
+    filename = f"produtos_{tenant_id}.xlsx"
+    return StreamingResponse(excel_file, 
+        media_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+        headers={"Content-Disposition": f"attachment; filename={filename}"}
+    )
+
+@router.get("/get-file/{retrieval_key}", tags=["Files"], dependencies=[Depends(get_current_user)])
+def get_file(retrieval_key: str, db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)